acri_controller/State x_in
float64 v
---
acri_controller/State x_out